.bazelci/config.yaml

```diff
@@ -201,18 +201,13 @@ tasks:
       - "--experimental_enable_aggregating_middleman=False"
   min_supported_version_examples:
     name: "Minimum Supported Version Examples"
-    bazel: "4.0.0"
```

Comment:
Technically bazel 4 is still under LTS support until the end of this year and so I'd rather that we just disable the failing tests and keep this at 4 as the rest of these changes only affect the examples / tests and don't change the ability of this ruleset to continue support v4.

Reply:
I restored 4.0 as the min version.

I'll try my hand at modifying the config.yaml to skip the offending tests, but I might come back asking for some help on how to do so. (If you happen to have a link that explains these bazelci config files, that'd be great; the only thing i've found is https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/README.md which doesn't explain many of the things i see or how to use them).

Reply:
OK, I think I figured it out.

Ready for another look